samples/timer/README.rst

```diff
@@ -0,0 +1,91 @@
+.. _timer_sample:
+
+TIMER
+#####
+
+.. contents::
+   :local:
+   :depth: 2
+
+The TIMER sample demonstrates how to control PMIC TIMERs using npmx drivers.
+
+Requirements
+************
+
+The sample supports the following development kits:
+
+.. table-from-sample-yaml::
+
+The sample also requires an nPM1300 EK.
+
+Overview
+********
+
+This sample uses the TIMER driver to confgure and run timer.
+The TIMER is configured to generate an interrupt after ~400 ms.
```

Comment:
```suggestion
The Timer driver is configured to generate an interrupt after ~400 ms.
```
**The Timer driver** or **The timer** (any)?

Reply:
Periphery